.changeset/tough-candles-travel.md

```diff
@@ -70,6 +70,10 @@ const handler = MakeswiftApiHandler(MAKESWIFT_SITE_API_KEY, {
             weight: '700',
             style: 'normal',
           },
+          {
+            weight: '900',
+            style: 'normal',
+          },
```

Comment:
Added a [missing weight](https://github.com/makeswift/makeswift/pull/909/files#diff-07b6dc10af104363083548a84c1cc33ac245470d5d56a13ee073ba101584d1e5R37) to the `Font` example.